src/pages/Rockets.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import fetchRocketsData from '../redux/rockets/rocketsActions';
+import Rocket from '../components/Rocket';
+
+const Rockets = () => {
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (!rockets) {
+      dispatch(fetchRocketsData());
+    }
+  }, [dispatch]);
+  return (
```

Comment:
- [ ] When I run the app using `npm start`, the RocketData doesn't appear, you have to remove this if condition 
or if you want to add a condition you can check on the length instead for example  `if (rockets.length === 0)` or  `if (!rockets.length)` 👍

https://user-images.githubusercontent.com/45231276/202425005-931b29c3-c5d1-413b-bd5d-a43762bacef0.mp4